crates/js_api/ARCHITECTURE.md

```
@@ -0,0 +1,193 @@
+**Overview**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)